mindbender/maya/lib.py

```diff
@@ -166,6 +166,7 @@ def export_alembic(nodes,
                    file,
                    frame_range=None,
                    uv_write=True,
+                   write_Visibility=True,
```

Comment:
I'm also thinking we avoid the bad naming convention used by the original `AbcExport` and prefix each of the 6 optional outputs with `write_`, including `uv_write`.

The 6 optionals being:

- uvWrite
- writeColorSets
- writeFaceSets
- writeVisibility
- writeCreases

Let's rename `uv_write` to `write_uv` and make `write_Visibility` into `write_visibility`. What do you think?